routes.jsx

```diff
@@ -3,10 +3,10 @@ var Route = require('react-router').Route;
 
 var routes = (
   <Route>
-    <Route name="thank-you" path="/thank-you" handler={require('./pages/thank-you.jsx')} />
-    <Route name="sequential" path="/" handler={require('./pages/sequential.jsx')} />
-    <Route name="share" path="/share" handler={require('./pages/share.jsx')} />
-    <Route name="paypal-donate-usd" path="/paypal-donate-usd" handler={require('./pages/paypal-donate-usd.jsx')} />
+    <Route name="thank-you" path="/thank-you/?" handler={require('./pages/thank-you.jsx')} />
+    <Route name="sequential" path="/?" handler={require('./pages/sequential.jsx')} />
```

Comment:
Might not, but I don't know how our server handles that, so I just put it anyway...